vmm/src/default_syscalls/filters.rs

```diff
@@ -6,25 +6,9 @@ use seccomp::{
     SeccompCmpOp::Eq, SeccompCondition as Cond, SeccompFilter, SeccompRule,
 };
 
-// Currently these variables are missing from rust libc:
-// https://github.com/rust-lang/libc/blob/master/src/unix/notbsd/linux/musl/b64/aarch64.rs
-// even though they are defined in musl libc:
-// https://git.musl-libc.org/cgit/musl/tree/arch/aarch64/bits/syscall.h.in.
-// Submitted issue in rust-lang: https://github.com/rust-lang/libc/issues/1348.
-#[allow(non_upper_case_globals)]
 #[cfg(target_arch = "aarch64")]
-mod libc_patch {
-    pub const SYS_fcntl: ::std::os::raw::c_long = 25;
-    pub const SYS_lseek: ::std::os::raw::c_long = 62;
-    pub const SYS_newfstatat: ::std::os::raw::c_long = 79;
-    pub const SYS_fstat: ::std::os::raw::c_long = 80;
-    pub const SYS_mmap: ::std::os::raw::c_long = 222;
-}
-
-#[cfg(target_arch = "aarch64")]
-use self::libc_patch::{SYS_fcntl, SYS_fstat, SYS_lseek, SYS_mmap, SYS_newfstatat};
-#[cfg(target_arch = "x86_64")]
-use libc::{SYS_fcntl, SYS_fstat, SYS_lseek, SYS_mmap};
+use libc::SYS_fstat;
```

Comment:
For consistency reasons, can we instead use their definition from libc without importing them by name?

All other syscalls are used as:
```
            allow_syscall(libc::SYS_accept4),
            allow_syscall(libc::SYS_brk),
            allow_syscall(libc::SYS_brk),
            allow_syscall(libc::SYS_clock_gettime),
            allow_syscall(libc::SYS_clock_gettime),
```